### PR TITLE
User view translation

### PR DESCRIPTION
**Pull request by jiteshjha**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] tests pass -- `rake test`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request are descriptively named
* [x] if possible, multiple commits squashed if they're smaller changes
* [x] reviewed/confirmed/tested by another contributor or maintainer
* [x] `development.sqlite.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by jiteshjha** [comment]
@jywarren Hello! I'm done with user view translations. What do you think?
Thank you!

**Comment by jywarren** [comment]
Hi, i'll be able to review on Friday (possibly before); perhaps you could
work on another PR in the meantime? Thanks!

On Wed, Jul 27, 2016 at 11:59 AM, Jitesh Kumar Jha <notifications@github.com

**Comment by jiteshjha** [comment]
Sure! :)

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -1,26 +1,26 @@
 <div class="col-md-3">
 
-<h3>What is Public Lab?</h3>
```
Comment (jywarren) [inline app/views/users/new.html.erb]: Hmm, this is a lot of individual lines -- is there a way to save whole HTML blocks, translated? That would make maintaining this, or changing it, much easier.

**Comment by jiteshjha** [comment]
Hello @jywarren , I have moved the entire div content as a html block inside the yaml file. Thanks!

**Comment by jywarren** [comment]
Looks great. Thanks!

**Comment by jiteshjha** [comment]
I rebased it on top of the latest master!

**Comment by jywarren** [comment]
Oh good, i'll try again. I had gotten 2 errors. Thanks a lot!

**Comment by jiteshjha** [comment]
Oh, yes. One of the test should be updated. Hold on a sec.

**Comment by jiteshjha** [comment]
Okay, all the tests pass now. What do you think?

**Comment by jywarren** [comment]
Running them...

**Comment by jywarren** [comment]
I still get an error: 

```
............DEPRECATION WARNING: Passing a template handler in the template name is deprecated. You can simply remove the handler name or pass render :handlers => [:erb] instead. (called from realtime at /home/warren/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/benchmark.rb:294)
DEPRECATION WARNING: Passing a template handler in the template name is deprecated. You can simply remove the handler name or pass render :handlers => [:erb] instead. (called from realtime at /home/warren/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/benchmark.rb:294)
..E
========================================================================================================================================================================
Error: test_should_choose_i18n_for_user/likes(I18nTest): ActiveRecord::StatementInvalid: SQLite3::SQLException: no such column: true: SELECT "node_selections"."user_id" AS t0_r0, "node_selections"."nid" AS t0_r1, "node_selections"."following" AS t0_r2, "node_selections"."liking" AS t0_r3, "node"."nid" AS t1_r0, "node"."vid" AS t1_r1, "node"."type" AS t1_r2, "node"."language" AS t1_r3, "node"."title" AS t1_r4, "node"."uid" AS t1_r5, "node"."status" AS t1_r6, "node"."created" AS t1_r7, "node"."changed" AS t1_r8, "node"."comment" AS t1_r9, "node"."promote" AS t1_r10, "node"."moderate" AS t1_r11, "node"."sticky" AS t1_r12, "node"."tnid" AS t1_r13, "node"."translate" AS t1_r14, "node"."cached_likes" AS t1_r15, "node"."drupal_comments_count" AS t1_r16, "node"."drupal_node_revisions_count" AS t1_r17, "node"."path" AS t1_r18, "node"."main_image_id" AS t1_r19 FROM "node_selections" LEFT OUTER JOIN "node" ON "node"."nid" = "node_selections"."nid" WHERE (status = 1 AND user_id = 2 AND liking = true AND (node.type = 'page' OR node.type = 'tool' OR node.type = 'place'))
app/models/drupal_users.rb:73:in `liked_pages'
app/controllers/users_controller.rb:134:in `likes'
test/integration/I18n_test.rb:236:in `block (2 levels) in <class:I18nTest>'
test/integration/I18n_test.rb:225:in `each'
test/integration/I18n_test.rb:225:in `block in <class:I18nTest>'
========================================================================================================================================================================
.....................
```

**Comment by jywarren** [comment]
Ah, could be an error for sqlite, you have to substitute `"true"` using `" = ?", true` as true is stored differently in each database.

**Comment by jywarren** [comment]
Yes, here, in `drupal_users.rb`:

```ruby
  def liked_pages
    NodeSelection.find(:all, :conditions => ["status = 1 AND user_id = ? AND liking = true AND (node.type = 'page' OR node.type = 'tool' OR node.type = 'place')",self.uid], :include => :drupal_node).collect(&:node).reverse
  end
```

**Comment by jywarren** [comment]
True cannot be specified as a string, you have to pass it in as a boolean value so Rails can adapt this to various database formats. 

Can you also look into the deprecation warning?

**Comment by jiteshjha** [comment]
Oh okay, I'm using MySQL for all my tests, so I didn't get this error.

**Comment by jiteshjha** [comment]
I removed the error and deprecation warnings -- Do you still get those?

**Comment by jywarren** [comment]
I still see: 

```
ActionView::Template::Error: SQLite3::SQLException: no such column: true: SELECT  DISTINCT "node".nid FROM "node" LEFT OUTER JOIN "node_selections" ON "node_selections"."nid" = "node"."nid" LEFT OUTER JOIN "community_tags" ON "community_tags"."nid" = "node"."nid" LEFT OUTER JOIN "term_data" ON "term_data"."tid" = "community_tags"."tid" LEFT OUTER JOIN "comments" ON "comments"."nid" = "node"."nid" WHERE (type = 'note' AND node_selections.liking = true AND node_selections.user_id = 2 AND node.status = 1) ORDER BY node_selections.nid DESC LIMIT 20 OFFSET 0
app/views/notes/_notes.html.erb:4:in `_app_views_notes__notes_html_erb___648671635_58292232'
```

**Comment by jywarren** [comment]
Here:

    app/models/drupal_users.rb:    DrupalNode.includes(:node_selections).where("type = 'note' AND node_selections.liking = true AND node_selections.user_id = ? AND node.sta

There was one more -- i don't think it was your code, actually, but your test triggered it? Anyhow I'm fixing and will add one commit. Thanks!

**Comment by jiteshjha** [comment]
That's odd, I haven't touched notes views yet, but I guess I'm calling a lot of 'profile' routes in these tests which have never been called before.

**Comment by jywarren** [comment]
Manually merged!

**Pull request merged**